nebari/template/stages/07-kubernetes-services/conda-store.tf

```diff
@@ -36,7 +36,7 @@ variable "conda-store-image" {
 variable "conda-store-image-tag" {
   description = "Version of conda-store to use"
   type        = string
-  default     = "v0.4.9"
+  default     = "v0.4.12"
```

Comment:
A few weeks ago, I added the default conda-store image tag as an input [here](https://github.com/nebari-dev/nebari/blob/bd777e6448b5e2d6339bc3d9ef35672163ae1945/nebari/constants.py#L11). This was part of the motivation for [this RFD](https://github.com/nebari-dev/nebari/issues/1569) :)

Reply:
Removing it works for me 👍